packages/math-input/src/components/tabbar/tabbar.tsx

```diff
@@ -62,4 +71,35 @@ function Tabbar(props: Props): React.ReactElement {
     );
 }
 
+// Custom focus hook to handle arrow key navigation for the TabBar for each ArrowKeyTabItem.
+function useArrowKeyFocus(size) {
```

Comment:
Open to pushback on this, but:
- If this logic is only used by this component, could it just be part of the component? Or is it just cleaner to split it out?
- If this logic could be used by other components, could we split it out into its own file?

Reply:
Ideally in the future we should work with WonderBlocks to use their future TabPanel which will have this functionality baked in. I see this as a temporary stop-gap to ensure users can navigate via arrow key until we can align on a longer term implementation.

But I can totally move this into a utility function as well 😊